system_metrics_collector/src/system_metrics_collector/linux_cpu_measurement_node.hpp

```diff
@@ -63,9 +63,9 @@ class LinuxCpuMeasurementNode : public system_metrics_collector::PeriodicMeasure
   LinuxCpuMeasurementNode(
     const std::string & name,
     const std::chrono::milliseconds measurement_period,
-    const std::string & topic,
+    const std::string & statistics_topic,
     const std::chrono::milliseconds publish_period =
-    PeriodicMeasurementNode::DEFAULT_PUBLISH_WINDOW);
+    PeriodicMeasurementNode::INVALID_PUBLISH_WINDOW);
 
```

Comment:
unrelated changes

Reply:
Removed the unrelated changes from this PR.